assets/src/components/viewHeader.tsx

```diff
@@ -45,7 +45,9 @@ const ViewHeader: ViewHeaderType = ({
           {backlinkTitle(backlinkToView)}
         </button>
       ) : null}
-      <div className="m-view-header__title">{title}</div>
+      <div className="m-view-header__title" role="heading" aria-level={2}>
```

Comment:
adding this role made it easier to look for in tests

Reply:
can we style a h2 to match and use it instead?

Reply:
I guess we'd also need a <h1> for the page header as well, but we don't have any places for that I can think of off the top of my head :/

Reply:
Oh yeah, I was wary of messing with the styling but looks like the styles for `m-view-header__title` are already pretty specific. So I can change to `<h2>` if that is more conventional even without an `<h1>` present